client/src/ui/molecules/language-toggle/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { Translation } from "../../../document/types";
+import { CATEGORY_LANGUAGE_TOGGLE, useGA } from "../../../ga-context";
```

Comment:
```suggestion
import { useGA } from "../../../ga-context";
```

That code, `ga-context.tsx` shouldn't have to know or be aware about users of it. There's no reason to define these constants there. We should fix this about `CATEGORY_LEARNING_SURVEY` too by the way. 

I know `CATEGORY_LANGUAGE_TOGGLE` is present in two files. But does it need to be the same? Because submitting the drop-down is very different in that it sets a persistent cookie. Whereas clicking "View in English" doesn't. 
So instead of calling both `change language` you call one `change language cookie` and the other `change language temporary`.